hypervisor/include/arch/x86/lapic.h

```diff
@@ -76,22 +76,67 @@ union apic_icr {
 };
 
 /**
- * @brief Save context of lapic
+ * @defgroup lpic_ext_apis LAPIC External Interfaces
```

Comment:
Nitpick: could we name this group `lapic_ext_apis` instead?

Reply:
> Nitpick: could we name this group `lapic_ext_apis` instead?

@gvancuts Thanks for pointing out the typo! I will fix it in later version.